en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -1,80 +1,39 @@
+This section describes how you can host the authentication endpoint on a different server outside the WSO2 Identity Server  (e.g., in a different Tomcat
+Server).
 
 ## Moving the authenticationendpoint from WSO2IS and hosting it on a separate web server
 
 !!! tip "Before you begin"
-    First, get a copy of
-    `         <IS_HOME>/repository/deployment/server/webapps        `
-    `         /authenticationendpoin.war        ` to
-    `         <WebApp_HOME>/        ` and unzip it. Make sure to get the
-    `         authenticationendpoin.war        ` after the WUM update and
-    NOT the extracted `         authentication        `
-    `         endpoint        ` in
-    `         <IS_HOME>/repository/deployment/server/webapps/        `
+    First, get a copy of the `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint.war` file to `<WEBAPP_HOME>/` and unzip it. Make sure to get the `authenticationendpoint.war` after the WUM update and NOT the extracted `authenticationendpoint.war` in the `<IS_HOME>/repository/deployment/server/webapps/` folder.
     
-
-1.  Copy the following .jar files from the
-    `           <IS_HOME>/repository/components/plugins/          `
-    directory to the
-    `           <WebApp_HOME>/authenticationendpoint/WEB-INF/lib          `
-    directory.
-
-    -   org.wso2.carbon.base_4.4.11.jar
-
-    -   org.wso2.carbon.identity.base_5.7.5.jar
-
-    -   org.wso2.carbon.ui_4.4.11.jar
-
-    -   org.wso2.carbon.identity.application.authentication.endpoint.util_5.7.5.jar
-
-    -   org.wso2.carbon.identity.core_5.7.5.jar
-
-    -   httpcore_4.3.3.wso2v1.jar
-
-    -   org.wso2.carbon.identity.user.registration.stub_5.7.5.jar 
-
-    -   axis2_1.6.1.wso2v20.jar
-
-    -   org.wso2.carbon.user.api_4.4.11.jar
-
-    -   opensaml_2.6.4.wso2v3.jar
-
-    -   org.wso2.carbon.utils_4.4.11.jar
-
-    -   jettison_1.3.4.wso2v1.jar
-
-    -   org.wso2.carbon.user.core_4.4.11.jar
-
-2.  Copy the following .jar files from the \<
-    `          IS_HOME>/lib/runtimes/cxf/         ` directory to the
-    `          <WebApp_HOME>/authenticationendpoint/WEB-INF/lib         `
+1.  Copy the following .jar files from the `<IS_HOME>/repository/components/plugins/` directory to the `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/lib` directory.
+
+    -   `org.wso2.carbon.base_4.4.11.jar`
+    -   `org.wso2.carbon.identity.base_5.7.5.jar`
+    -   `org.wso2.carbon.ui_4.4.11.jar`
+    -   `org.wso2.carbon.identity.application.authentication.endpoint.util_5.7.5.jar`
+    -   `org.wso2.carbon.identity.core_5.7.5.jar`
+    -   `httpcore_4.3.3.wso2v1.jar`
+    -   `org.wso2.carbon.identity.user.registration.stub_5.7.5.jar` 
+    -   `axis2_1.6.1.wso2v20.jar`
+    -   `org.wso2.carbon.user.api_4.4.11.jar`
+    -   `opensaml_2.6.4.wso2v3.jar`
+    -   `org.wso2.carbon.utils_4.4.11.jar`
+    -   `jettison_1.3.4.wso2v1.jar`
+    -   `org.wso2.carbon.user.core_4.4.11.jar`
```

Comment:
Seems there are .jars missing in this list. Refer https://github.com/wso2/samples-is/issues/250, https://github.com/wso2/samples-is/blob/master/host-endpoints-externally/setup-authentication-endpoint.sh to find references.